lib/razor/data/node.rb

```diff
@@ -181,6 +183,26 @@ def registered?
       ! facts.nil?
     end
 
+    #Determine if this machine should be protected as a new machine based
+    #on the value of the config param 'protect_new_nodes' and 'invert_protect_new_nodes_for_subnets'
+    def protect?
+      protected = Razor.config['protect_new_nodes'] || false
+      override_subnets = Razor.config['invert_protect_new_nodes_for_subnets'] || []
+
+      if self.metadata['ip']
```

Comment:
Is this metadata value set so consistently that it can reliably factor into deciding whether to provision a node? Where is this {{ip}} node metadata value coming from, if not the task's {{store_ip}} function when the node has just completed installation? I can't see a picture here of how this works when {{protect_new_nodes}} is false.